service/src/main/resources/changelog/changesets/20220823_run.yaml

```diff
@@ -49,3 +49,9 @@ databaseChangeLog:
                   type: timestamp
                   constraints:
                     nullable: false
+              - column:
+                  name: error_messages
+                  type: varchar(1000)
+                  constraints:
+                    nullable: true
```

Comment:
Fine for now... at some point we will need to get into the habit of making changesets instead of "changing history" for these database updates... but it's fine for now.

Reply:
Does that mean, automatic vs manual?